openfisca_france/model/prestations/education.py

```diff
@@ -302,3 +303,20 @@ class scolarite(Variable):
     entity = Individu
     label = "Scolarité de l'enfant : collège, lycée..."
     definition_period = MONTH
+
+
+class TypesEtablissementScolaire(Enum):
```

Comment:
Le terme privilégié semble être « statut » plutôt que « type », voir par exemple [ce document](https://www.onisep.fr/Parents/Comprendre-l-organisation-des-etudes-superieures-pour-echanger-avec-votre-enfant/Les-etablissements-prives-quels-statuts) de l'Onisep.

```suggestion
class StatutsEtablissementScolaire(Enum):
```